.travis.yml

```diff
@@ -4,8 +4,7 @@ language: groovy
 
 jdk:
   - oraclejdk8

```

Comment:
>  People using this plugin might not necessarily be using Java 8,

Not sure if this still makes any sense. One can't simply download  a recent Oracle JDK 7 without paying to Oracle anymore.